Library/Homebrew/cleanup.rb

```diff
@@ -1,6 +1,71 @@
 require "utils/bottles"
 require "formula"
 
+module CleanupRefinement
+  refine Pathname do
+    def incomplete?
+      extname.end_with?(".incomplete")
+    end
+
+    def nested_cache?
+      directory? && ["glide_home", "java_cache", "npm_cache"].include?(basename.to_s)
+    end
+
+    def prune?(days)
+      return false unless days
+      return true if days.zero?
+
+      # TODO: Replace with ActiveSupport's `.days.ago`.
```

Comment:
Do you want this to block the PR?

Reply:
No, this is just so we don't forget to use it when we have it.